misk-hibernate/src/main/kotlin/misk/hibernate/SessionFactoryService.kt

```diff
@@ -81,8 +81,12 @@ internal class SessionFactoryService(
       applySetting(AvailableSettings.DRIVER, config.type.driverClassName)
       applySetting("hibernate.hikari.driverClassName", config.type.driverClassName)
       applySetting(AvailableSettings.URL, config.type.buildJdbcUrl(config, environment))
-      applySetting(AvailableSettings.USER, config.username)
-      applySetting(AvailableSettings.PASS, config.password)
+      if (config.username != null) {
```

Comment:
should we add some validation that you need to either set username/password or provide the mTLS creds?